components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dto/OAuth2AccessTokenRespDTO.java

```diff
@@ -189,6 +191,24 @@ public Map<String, String> getParameters() {
         return Collections.unmodifiableMap(getParameterMap());
     }
 
+    /**
+     * Get error parameter map
+     *
+     * @return a map of error parameters
+     */
+    public Map<String, Object> getErrorParameterMap() {
+        return errorParameterMap;
```

Comment:
Would be good to return an unmodifiable map and introduce a seperate method to add a new error param

Reply:
Fixed.